src/sonic-daemon-base/sonic_daemon_base/daemon_base.py

```diff
@@ -81,6 +81,7 @@ def get_namespaces():
         # For mulit Asic platform the list if valid namespaces other than the 
         # default linux host namespace.
         if multiAsic.is_multi_asic():
+            from swsscommon import swsscommon
```

Comment:
Why move this?

Reply:
Here too it was modified similary as done in db_connect